### PR TITLE
[platforms] Copy edit and clarity pass

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

File: docs/resources/add-ins-differences.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 description: 'The behavior and API differences between Office Scripts and Office Add-ins.'
-ms.date: 06/01/2020
+ms.date: 06/02/2021
 localization_priority: Normal
 ---
 
 # Differences between Office Scripts and Office Add-ins
 
-Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook a JavaScript API. However, the Office Scripts APIs are a specialized, synchronous version of the Office JavaScript API.
+Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook through JavaScript APIs. However, Office Scripts are designed to be quickly made by anyone looking to improve their workflow. Whereas Office Add-ins are larger-scale integrations for an Office host application.
```
Comment (davidchesnut) [inline docs/resources/add-ins-differences.md:10]: Seems odd to follow up a header about "differences" with how much is in common. How about this?
```suggestion
Understand the differences between Office Scripts and Office Add-ins to know when to use each one. Office Scripts are designed to be quickly made by anyone looking to improve their workflow. Office Add-ins integrate with the Office UI for a more interactive experience through ribbon buttons and task panes. Office Add-ins can also expand built-in Excel functions by providing custom functions.
```

File: docs/resources/add-ins-differences.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
-Office Scripts run to completion with a manual button press or as a step in [Power Automate](https://flow.microsoft.com/), whereas Office Add-ins persist while their task panes are open. This means the add-ins can maintain state during a session, whereas Office Scripts do not maintain an internal state between runs. If you find that your Excel extension needs to exceed the scripting platform's capabilities, visit the [Office Add-ins documentation](/office/dev/add-ins) to learn more about Office Add-ins.
+Office Scripts run to completion with a manual button press or as a step in [Power Automate](https://flow.microsoft.com/), whereas Office Add-ins persist while their task panes are open. This means the add-ins maintain state during a session, whereas Office Scripts don't maintain an internal state between runs. If you find that your Excel extension needs exceed the scripting platform's capabilities, visit the [Office Add-ins documentation](/office/dev/add-ins) to learn more about Office Add-ins.
```
Comment (davidchesnut) [inline docs/resources/add-ins-differences.md:14]:
```suggestion
Office Scripts run to completion with a manual button press or as a step in [Power Automate](https://flow.microsoft.com/), whereas Office Add-ins continue running depending on how they are configured. For example, you can configure an Office Add-in to continue running even when its task pane is closed. This means that Office Add-ins maintain state during a session, whereas Office Scripts don't maintain an internal state between runs. If the solution you are building requires a maintained state, you should visit the [Office Add-ins documentation](/office/dev/add-ins) to learn more about Office Add-ins.
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Would UI differences be another section to list?

**Comment by AlexJerabek** [comment]
Thanks @davidchesnut. I agree that the UI could be another difference. Maybe we can add that to the backlog. As for now, I applied your changes. Is there anything else I should take a look at?

**Review by davidchesnut — APPROVED** [review]
Looks great!

**Pull request merged**